lib/__tests__/standalone-fix.test.mjs

```diff
@@ -289,8 +340,60 @@ it('one rule being disabled and another still autofixing', async () => {
 		),
 	).toBe(true);
 
-	expect(result.output).toBe(`
+	expect(result.code).toBe(`
 		a, a {
 			color: #fff;
 		}`);
+	expect(getCleanOutput(result.output)).toBe(stripIndent`
+		test.css
+		 2:3  ×  Unexpected duplicate selector "a", first used at line 2  no-duplicate-selectors
+		
+		1 problem (1 error, 0 warnings)
+	`);
+});
+
+it("doesn't return the fixed code if the fix option is false", async () => {
+	const result = await standalone({
+		code: 'a { color: #ffffff; }',
+		codeFilename: 'test.css',
+		config: {
+			rules: {
+				'color-hex-length': 'short',
+			},
+		},
+		fix: false,
+		formatter: 'string',
+	});
+
+	expect(result.code).toBeUndefined();
+	expect(getCleanOutput(result.output)).toBe(stripIndent`
+		test.css
+		 1:12  ×  Expected \"#ffffff\" to be \"#fff\"  color-hex-length
```

Comment:
question

Is escaping double quotes necessary?

Reply:
Good catch! I copied the text from VS Code terminal and maybe the terminal added those backslashes for me. 😂 

Updated here: 0f5f531ffe54bba6dfb44ec0e62ab99538b26b83